alphalens/tears.py

```diff
@@ -417,6 +419,24 @@ def create_returns_tear_sheet(factor_data,
         plt.show()
         gf.close()
 
+    if factor_returns.index.tzinfo is None:
+        warnings.warn("to do a risk factor performance attribution,"
+                      "you must pass in a tz-aware Series")
+        return
```

Comment:
Instead of returning, I believe the analysis could continue skipping return decomposition